FitGoal/FitGoal/RoutinePlayerPageViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  RoutinePlayerPageViewController.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 13/5/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class RoutinePlayerPageViewController: UIPageViewController, UIPageViewControllerDelegate, UIPageViewControllerDataSource {
+    
+    private var currentIndex = 0
+    
+    private let routine: [Exercise]
+    private let firstExerciseIndex: Int
+    
+    private var exercisePlayerViewControllers: [UIViewController] = []
+    
+    private let stopButton: UIButton = {
+        let button = UIButton()
+        button.setAttributedTitle("stop".uppercased().formattedText(font: "Roboto-Light", size: 15, color: .white, kern: 0.18), for: .normal)
+        return button
+    }()
+    
+    init(firstExerciseIndex: Int, routine: [Exercise]) {
+        self.firstExerciseIndex = firstExerciseIndex
```

Comment:
It seems you don't need the `self.firstExerciseIndex` since you only use if for assigning its value to the `currentIndex` field. You can assign that value directly here using `self.currentIndex = firstExerciseIndex`

Reply:
Actually, it seems the value of `currentIndex` is never used, so you can remove that field and continue using `self.firstExerciseIndex` as you are doing now.